ruby/README.md

```diff
@@ -247,10 +247,44 @@ A developer should be able to view any `it` block in isolation --without its
 DSL methods, it's probably fine.
 
 - Avoid `let`, and `subject` (prefer factory methods)
+- Avoid unnecessary parenthesis in assertions to reduce noise
+
+  ```rb
+  # Good
+  expect(x).to eq y
```

Comment:
Slack poll: https://codeclimate.slack.com/archives/dev/p1486663111011158

So far `expect(x).to eq(y)` is winning.